website/templates/project/project_base.mako

```diff
@@ -14,6 +14,11 @@
 
 <%include file="project_header.mako"/>
 
+% if status and node:
+    ## WARNING: Need new heading alert messages? They must be added at both here and base.mako
```

Comment:
Is the warning necessary? If a new alert needs to be added, it only needs to be added to `alert.mako`, correct?